src/net.cpp

```diff
@@ -2811,18 +2811,17 @@ void CConnman::PushMessage(CNode* pnode, CSerializedNetMsg&& msg)
         pnode->mapSendBytesPerMsgCmd[msg.m_type] += nTotalSize;
         pnode->nSendSize += nTotalSize;
 
-        if (pnode->nSendSize > nSendBufferMaxSize)
-            pnode->fPauseSend = true;
+        if (pnode->nSendSize > nSendBufferMaxSize) pnode->fPauseSend = true;
```

Comment:
Please avoid change unrelated lines (unless there are good reasons doing so). It makes it harder to reason about the actual changes if one includes unrelated whitespace changes in the same commit :)

Applies also below and throughout this PR.

Reply:
I made these changes in response to [this comment](https://github.com/bitcoin/bitcoin/pull/19509#discussion_r457072901).  As far as I can tell they're all in the first commit 84a79d61cef5c9386597869adccda4aa5992cc96 (which only contains the unrelated changes), and they don't show up at all in `git range-diff master 9898fa1 cbb154e`.  i.e. on my machine I haven't mixed any real changes with whitespace changes.

Are you seeing something different?  Is there a better way I could have done this?

Reply:
This seems reasonable. Although the general advice is to not fix style on lines that you're not otherwise touching, if it's all contained within a single cleanup commit, then I think that's also fine.

Reply:
Might you add something to this affect to the commit title so people know that? I didn't get that from reading "Clean PushMessage and ProcessMessages". Maybe "Whitespace only changes in PushMessage and ProcessMessages"